tsdb/index/tsi1/log_file.go

```diff
@@ -544,21 +544,22 @@ func (f *LogFile) execSeriesEntry(e *LogEntry) {
 	// Generate key & series, if not exists.
 	// TODO(edd) use a pool of buffers?
 	key := AppendSeriesKey(nil, e.Name, e.Tags)
-	serie := mm.series[string(key)]
+	keyStr := string(key)
+	serie := mm.series[keyStr]
```

Comment:
@mark-rushakoff that compiler optimisation is moot here because we have to allocate `keyStr` anyway (since further down we save it in multiple maps).

In this case, since we already need to allocate `keyStr`, `serie := mm.series[keyStr]` and `serie := mm.series[string(key)]` are equivalent.